### ISSUE TITLE
Won't produce `.html` files since after 0.3.8

### ISSUE THREAD

**Issue by lolmaus** [zont/gulp-usemin#112]
Hi!

We've been using `gulp-usemin@0.3.8` happily. When we upgraded to `0.3.11`, `.html` files stopped producing.

The task would finish fine without errors, but no `.html` files in the output dir.

We had to lock `0.3.8`.

Our task:

```coffee
gulp.task "usemin", ->
  gulp
    .src "./*.html"
    .pipe usemin
      html: [
        minifyHtml empty: true
      ]
      js: [
        ngAnnotate()
        uglify()
        rev()
      ]
    .pipe gulp.dest './.tmp_deploy/dist'
```

**Comment by johannesjo** [comment]
Same issue here.

**Comment by pizthewiz** [comment]
Could this be #91?

**Comment by chrxs** [comment]
+1 same here

**Comment by hellfrom** [comment]
+1

**Comment by lihaibh** [comment]
+1 :( annoying

**Comment by chenkehxx** [comment]
+1

**Comment by daviskoh** [comment]
+1

**Comment by jorgeriv** [comment]
+1

**Comment by facupedrazzini** [comment]
+1

**Comment by zont** [comment]
Fixed in 0.3.14
